Sources/MongoSwift/Operations/EstimatedDocumentCountOperation.swift

```diff
@@ -24,7 +24,7 @@ public struct EstimatedDocumentCountOptions: Codable {
         self.readPreference = readPreference
     }
 
-    private enum CodingKeys: String, CodingKey {
+    internal enum CodingKeys: String, CodingKey, CaseIterable {
```

Comment:
Is there a reason we want it to be internal now?

Reply:
good question. I am using this type to generate a set of "known arguments" for estimatedDocumentCount. the unified test runner requires we error if any arguments specified for an operation.
to access this type from the tests, `internal` is necessary (when `private`, it was only accessible within `EstimatedDocumentCountOptions` itself)